meta-mbl-distro/classes/mbl-partitions.bbclass

```diff
@@ -227,23 +246,26 @@ MBL_LOG_LABEL ?= "log"
 MBL_LOG_DEFAULT_SIZE_MiB ?= "128"
 MBL_LOG_MOUNT_OPTS ?= "rw,noexec,nodev,nosuid,async"
 MBL_PARTITION_NAMES += "LOG"
-
+MBL_PARTITION_VARS += "\
+MBL_LOG_MOUNT_POINT MBL_LOG_LABEL MBL_LOG_DEFAULT_SIZE_MiB MBL_LOG_MOUNT_OPTS"
 # ------------------------------------------------------------------------------
 # Scratch partition (file system; single bank)
 # ------------------------------------------------------------------------------
 MBL_SCRATCH_MOUNT_POINT = "${MBL_SCRATCH_DIR}"
 MBL_SCRATCH_LABEL ?= "scratch"
 MBL_SCRATCH_DEFAULT_SIZE_MiB ?= "500"
 MBL_PARTITION_NAMES += "SCRATCH"
-
+MBL_PARTITION_VARS += "\
+MBL_SCRATCH_MOUNT_POINT MBL_SCRATCH_LABEL MBL_SCRATCH_DEFAULT_SIZE_MiB"
 # ------------------------------------------------------------------------------
 # Home partition (file system; single bank)
 # ------------------------------------------------------------------------------
 MBL_HOME_MOUNT_POINT = "${MBL_HOME_DIR}"
 MBL_HOME_LABEL ?= "home"
 MBL_HOME_DEFAULT_SIZE_MiB ?= "450"
 MBL_PARTITION_NAMES += "HOME"
-
+MBL_PARTITION_VARS += "\
+MBL_HOME_MOUNT_POINT MBL_HOME_LABEL MBL_HOME_DEFAULT_SIZE_MiB"
```

Comment:
I think `MBL_PARTITION_VARS` is missing a bunch of variables that are only set in the Python section below.

I suggest just creating `MBL_PARTITION_VARS` in the Python section.

Reply:
Will do.

Reply:
I see this was done in https://github.com/ARMmbed/meta-mbl/pull/657/commits/ecf4db4b2ffd7350ef86cfa170c8e4dc14ecf8bf.

Thanks :-)